app/js/disease-associations-table-directive.js

```diff
@@ -54,37 +48,57 @@ angular.module('cttvDirectives')
-        {name:"", title:cttvDictionary.ASSOCIATION_SCORE},
+        {name: "", title: cttvDictionary.TARGET_SYMBOL},
+        {name: "", title: cttvDictionary.ENSEMBL_ID},
+        {name: "", title: cttvDictionary.ASSOCIATION_SCORE},
         // here are the datatypes:
-        {name:cttvConsts.datatypes.GENETIC_ASSOCIATION, title:cttvDictionary[cttvConsts.datatypes.GENETIC_ASSOCIATION.toUpperCase()]},
-        {name:cttvConsts.datatypes.SOMATIC_MUTATION, title:cttvDictionary[cttvConsts.datatypes.SOMATIC_MUTATION.toUpperCase()]},
-        {name:cttvConsts.datatypes.KNOWN_DRUG, title:cttvDictionary[cttvConsts.datatypes.KNOWN_DRUG.toUpperCase()]},
-        {name:cttvConsts.datatypes.AFFECTED_PATHWAY, title:cttvDictionary[cttvConsts.datatypes.AFFECTED_PATHWAY.toUpperCase()]},
-        {name:cttvConsts.datatypes.RNA_EXPRESSION, title:cttvDictionary[cttvConsts.datatypes.RNA_EXPRESSION.toUpperCase()]},
-        {name:cttvConsts.datatypes.LITERATURE, title:cttvDictionary[cttvConsts.datatypes.LITERATURE.toUpperCase()]},
-        {name:cttvConsts.datatypes.ANIMAL_MODEL, title:cttvDictionary[cttvConsts.datatypes.ANIMAL_MODEL.toUpperCase()]},
+        {
+            name: cttvConsts.datatypes.GENETIC_ASSOCIATION,
+            title: cttvDictionary[cttvConsts.datatypes.GENETIC_ASSOCIATION.toUpperCase()]
+        },
+        {
+            name: cttvConsts.datatypes.SOMATIC_MUTATION,
+            title: cttvDictionary[cttvConsts.datatypes.SOMATIC_MUTATION.toUpperCase()]
+        },
+        {
+            name: cttvConsts.datatypes.KNOWN_DRUG,
+            title: cttvDictionary[cttvConsts.datatypes.KNOWN_DRUG.toUpperCase()]
+        },
+        {
+            name: cttvConsts.datatypes.AFFECTED_PATHWAY,
+            title: cttvDictionary[cttvConsts.datatypes.AFFECTED_PATHWAY.toUpperCase()]
+        },
+        {
+            name: cttvConsts.datatypes.RNA_EXPRESSION,
+            title: cttvDictionary[cttvConsts.datatypes.RNA_EXPRESSION.toUpperCase()]
+        },
+        {
+            name: cttvConsts.datatypes.LITERATURE,
+            title: cttvDictionary[cttvConsts.datatypes.LITERATURE.toUpperCase()]
+        },
+        {
+            name: cttvConsts.datatypes.ANIMAL_MODEL,
+            title: cttvDictionary[cttvConsts.datatypes.ANIMAL_MODEL.toUpperCase()]
+        },
         // empty col for sorting by total score (sum)
-        {name:"", title:"total score"},
+        {name: "", title: "total score"},
         // empty col for the gene name
-        {name:"", title:cttvDictionary.TARGET_NAME}
+        {name: "", title: cttvDictionary.TARGET_NAME}
     ];
 
 
     /*
     Setup the table cols and return the DT object
     */
-    var setupTable = function(table, disease, filename, download, stt){
-        $log.log("setupTable()");
-
+    var setupTable = function (table, disease, target, filename, excludedTargetList, download, stt) {
         stt = stt || {};
 
         var t = $(table).DataTable({
+            "destroy": true,
```

Comment:
Don't destroy the table, this is very inefficient. It is better to use something like table.ajax.reload() when new data is available